OracleTuxedo/salt/samples/bankapp/start.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+
+cd /u01/data/bankapp/ || return 1
+
+APPDIR=$(pwd)
+export APPDIR
+
+UNAME1=$(uname -n)
+TUX_UID=$(id -u)
+TUX_GID=$(id -g)
+
+
+# clean up from any previous run
+tmshutdown -y &>/dev/null 
+rm -f TLOG GWTLOG tuxconfig saltconfig bankdl1 bankdl2 bankdl3 ULOG.*
+rm -f "${SHUTDOWN_MARKER_FILE}"
```

Comment:
```suggestion
rm -f "$SHUTDOWN_MARKER_FILE"
```

Where is `$SHUTDOWN_MARKER_FILE` set?

Reply:
At line 46 in start.sh and which is inherited by this stop script due to the `source "${APPDIR}/bankvar.new` at line 13 in this file.

Reply:
This _is_ `start.sh` so you're trying to use this variable 27 lines too early.

Reply:
Good point. I moved the deletion of the shutdown marker file a little later after setting of the env var. 

Though in theory, the original script would have also worked fine because the first time the start.sh is run, there is no marker file and so nothing to delete anyways. And subsequent runs of start.sh will have the SHUTDOWN_MARKER_FILE  env var already set before even the start.sh script starts, courtesy the "export" of that env var done in the prior runs of the start.sh.